Configuration/AlCa/python/autoCond.py

```diff
@@ -136,5 +136,5 @@
 autoCond['upgradePLS1']      = ( autoCond['run2_mc'] )
 autoCond['upgradePLS150ns']  = ( autoCond['run2_mc_50ns'] )
 autoCond['upgrade2017']      = ( autoCond['phase1_2017_design'] )
-autoCond['upgrade2021']      = ( autoCond['phase1_2021_design'] )
```

Comment:
Wouldn't it be better to retain `upgrade2021`? Just add `upgrade2022` and keep both so anything using `upgrade2021` will still work?

Reply:
Sure I can do that